jmclient/setup.py

```diff
@@ -12,6 +12,6 @@
       install_requires=['joinmarketbase==0.9.10dev', 'mnemonic==0.20',
                         'argon2_cffi==21.3.0', 'bencoder.pyx==3.0.1',
                         'pyaes==1.6.1', 'klein==20.6.0', 'pyjwt==2.4.0',
-                        'autobahn==20.12.3'],
+                        'autobahn==20.12.3', 'werkzeug=2.2.0'],
```

Comment:
Must use `==` instead of `=`, will fail otherwise

```
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [3 lines of output]
      error in joinmarketclient setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Expected end or semicolon (after name and no valid version specifier)
          werkzeug=2.2.0
                  ^
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.
```

Reply:
Fixed in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1485/commits/4b2cbe9b34405dc1536169ee2b1553743c30b49c

Reply:
One mistake easy to overlook :)